src/gdk/gdk_posix.c

```diff
@@ -1041,6 +1045,44 @@ MT_sleep_ms(unsigned int ms)
 
 #endif
 
-#ifdef HAVE_EMBEDDED_R
+#if defined(HAVE_EMBEDDED_R) || defined(HAVE_EMBEDDED_JAVA)
 char* monetdb_lib_path = NULL;
 #endif
+
+#ifdef HAVE_EMBEDDED_JAVA
+void freeMonetDB5LibraryPath(void) {
+	if(monetdb_lib_path != NULL) {
+		GDKfree(monetdb_lib_path);
+		monetdb_lib_path = NULL;
+	}
+}
+
+int setMonetDB5LibraryPath(const char* path) {
+	if(monetdb_lib_path != NULL) {
+		freeMonetDB5LibraryPath();
+	}
+	monetdb_lib_path = GDKstrdup(path);
+	return monetdb_lib_path != NULL ? 1 : 0;
+}
+
+#ifdef NATIVE_WIN32
+/* on Windows compilation of MonetDBJavaLite, this function was still missing */
+/* adapted from https://llvm.org/svn/llvm-project/compiler-rt/branches/release_30/lib/popcountdi2.c */
+int __popcountdi2(unsigned long long a) {
```

Comment:
what is it used for?

Reply:
That symbol was missing in the Windows DLL.

Reply:
Fixed it. Using a compiler flag.